src/Products/urban/Applicant.py

```diff
@@ -307,7 +307,7 @@ def _getNameSignaletic(
         self, short, linebyline, reverse=False, invertnames=False, withtitle=True
     ):
         title = self.getPersonTitleValue(short, linebyline, reverse)
-        name1 = self.getName1().decode("utf-8")
```

Comment:
For the future, this must be an option, to be compatible with both Classic & Liege